### DISCUSSION TITLE
Template question involving environment variable

### DISCUSSION THREAD

**Discussion by mrolli** [JanDeDobbeleer/oh-my-posh#1700] [Troubleshoot] [answered]
As a Go noob I'm having a hard time getting my head around the templating. While I can read and understand the templates in docs, i.e. the examples for the git segment, I can't extrapolate and get my things to run.

I'm trying to get a simple text segment to run involving an environment variable and output text based on this variable. The following bash snippet illustrates what I want to achieve:

    [ "$CONDA_PROMPT_MODIFIER" != "false" ] && echo $CONDA_PROMPT_MODIFIER

Works like a charm and does exactly what I want. Now I'm trying to do the very same in the text segment using a template:

        {
          "foreground": "#458588",
          "properties": {
            "template": "{{ if (ne .Env.CONDA_PROMPT_MODIFIER 'false') }}{{ .Env.CONDA_PROMPT_MODIFIER }}{{ end }}"
          },
          "style": "plain",
          "type": "text"
        },

This attempt ends in a nice blue "invalid template text". What am I doing wrong here?

**Comment by JanDeDobbeleer** [comment, marked as answer]
Try `if ne .Env.CONDA_PROMPT_MODIFIER \"false\"`

**Reply by mrolli** [reply]
That's it! Many thanks.

Lesson learned: Always enclose strings in double quotes and escape them if needed to.
